esrally/mechanic/telemetry.py

```diff
@@ -776,17 +777,21 @@ def _store_merge_times(self, merge_times):
 
 
 class DiskIo(InternalTelemetryDevice):
+    human_name = "Disk IO"
```

Comment:
when we get rid of the console message (see my comment below), this will be unused and we can remove it as well.